.github/workflows/quickstart.yaml

```diff
@@ -0,0 +1,58 @@
+name: Build and Deploy
+
+on: [push, pull_request]
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: 1.21
+
+      - name: Install dependencies
+        run: |
+          # Install kind
+          curl -sLo kind "$(curl -sL https://api.github.com/repos/kubernetes-sigs/kind/releases/latest | jq -r '[.assets[] | select(.name == "kind-linux-amd64")] | first | .browser_download_url')"
+          chmod +x kind
+          sudo mv kind /bin/
+          
+          # Install kubectl
+          curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl"
+          chmod +x kubectl
+          sudo mv kubectl /bin/
+          
+      - name: Build Docker image
+        run: |
+          sudo -E docker build -t checkpoint-restore-operator:ci .
+
+      - name: Delete existing Kubernetes cluster (if any)
+        run: |
+          kind delete cluster || true
+
+      - name: Create Kubernetes cluster
+        run: |
+          kind create cluster
+          kind export kubeconfig
+
+      - name: Load Docker image into kind cluster
+        run: |
+          kind load docker-image checkpoint-restore-operator:ci
+
+      - name: Deploy to Kubernetes
+        run: |
+          make install
+          make deploy IMG=checkpoint-restore-operator:ci
+
+      - name: Add execute permissions to wait_for_deployment.sh
+        run: chmod +x ./test/wait_for_deployment.sh
```

Comment:
Would it be possible to commit this file with execute permissions, e.g., `chmod +x ... && git commit -a --amend`?